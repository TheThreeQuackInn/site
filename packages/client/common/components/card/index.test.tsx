import React from 'react';
import {create} from 'react-test-renderer';
import Card from './';

describe('Card', () => {
    it('renders correctly', () => {
        const component = create(
            <Card title="Testing" subTitle={<span>Subtitle</span>}>
                <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum ante mauris, dapibus at
                    tristique non, hendrerit at diam. Curabitur volutpat, ipsum ac imperdiet tincidunt, felis ipsum
                    tincidunt lorem, vel tristique purus lacus eget ipsum. Sed gravida mattis volutpat. Nulla lacinia
                    nisi egestas justo rhoncus, eget feugiat neque luctus. Vivamus quis turpis orci. Mauris vel nisi
                    quis purus dictum placerat in eu mi. Pellentesque aliquet leo vitae justo laoreet tempor. In
                    sagittis eleifend tempor. Cras tincidunt ultrices lacus quis finibus. Mauris mauris nulla, pharetra
                    in dolor quis, varius facilisis neque. Vestibulum tempus fermentum diam at tristique. Aliquam congue
                    laoreet enim, a aliquam urna feugiat non.
                </p>
            </Card>,
        );
        expect(component).toMatchSnapshot();
    });
});
