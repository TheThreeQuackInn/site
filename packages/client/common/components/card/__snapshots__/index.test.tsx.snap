// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="rounded-lg bg-white dark:bg-gray-800 shadow-lg md:shadow-xl relative overflow-hidden"
>
  <div
    className="px-3 pt-2 pb-3 md:p-5 relative box-border w-full"
  >
    <p
      className="font-bold text-xl mb-2 dark:text-white"
    >
      Testing
    </p>
    <div
      className="dark:text-white text-gray-700"
    >
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum ante mauris, dapibus at tristique non, hendrerit at diam. Curabitur volutpat, ipsum ac imperdiet tincidunt, felis ipsum tincidunt lorem, vel tristique purus lacus eget ipsum. Sed gravida mattis volutpat. Nulla lacinia nisi egestas justo rhoncus, eget feugiat neque luctus. Vivamus quis turpis orci. Mauris vel nisi quis purus dictum placerat in eu mi. Pellentesque aliquet leo vitae justo laoreet tempor. In sagittis eleifend tempor. Cras tincidunt ultrices lacus quis finibus. Mauris mauris nulla, pharetra in dolor quis, varius facilisis neque. Vestibulum tempus fermentum diam at tristique. Aliquam congue laoreet enim, a aliquam urna feugiat non.
      </p>
    </div>
  </div>
</div>
`;
